ocs_ci/ocs/defaults.py

```diff
@@ -17,7 +17,7 @@
 
 INSTALLER_VERSION = '4.1.4'
 CLIENT_VERSION = INSTALLER_VERSION
-ROOK_CLUSTER_NAMESPACE = 'rook-ceph'
+ROOK_CLUSTER_NAMESPACE = 'openshift-storage'
```

Comment:
Any reason we'd like to revert to openshift-storage? 
I think it'll be better to keep it until we fix the pod disappearance bug

Reply:
same